packages/@okta/vuepress-theme-prose/util/api.service.js

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+export class Api {
+  baseheaders = {
+    'content-type': 'application/x-www-form-urlencoded'
+  }
```

Comment:
We may only want these on POST -- sending content-type for GETs doesn't make sense